documentation/templates/tutorial-template-intro.asciidoc

```diff
@@ -35,12 +35,13 @@ As declarative layouts are a robust and maintainable way for defining views, htt
 The template API has limitations that you need to be aware of before using it.
 
 * The framework considers any attribute other than `theme` or one of the https://www.w3.org/TR/html52/dom.html#global-attributes[HTML 5 global attributes] for server-side initialization. However, if the attribute value is not a constant, that is, it contains bindings such as `${...}` and `{{...}}`, it is not parsed and the property is not initialized on the server side. Only attributes are parsed: properties defined by nested elements, such as grid columns and items, are not parsed from the template but must be initialized on the server side.
-* You can add components and a DOM to a template structure, but you cannot remove anything present in the original template. However, overriding component properties do work.
+* You can add components and elements to a template structure, but you cannot remove anything present in the original template. However, overriding component properties do work.
+* For `LitTemplate`, you cannot configure the column renderers for `Grid`, `TreeGrid` or `GridPro` in HTML. The column configuration and driving of data needs to happen from Java to the `@Id`-mapped component. This is different than with `PolymerTemplate`.
 * Calling `setText` from Java on a template-based component causes removal of child elements. For example, if the template is `<div id="myDiv">Some text<vaadin-button id="myButton">My Button</vaadin-button></div>`, calling `myDiv.setText` will cause the `myButton` element to be removed. To keep children, instead wrap the text content in a separate child alongside `myButton` and call `setText` on that element: `<div id="myDiv"><span id="myTest">Some text</span><vaadin-button id="myButton">My Button</vaadin-button></div>`.
 
 == Correctly Separating Structure and Logic
 Strive to only define things that do not change in the template and leave all dynamic parts to Java. In the template you can, for example, have attributes that never change, captions for components like `Button` and `TextField` that always stay the same, or styling for the view. If, for example, a `Button` should change the caption depending on user interaction, then leave it empty in the template and define the initial value in Java.
 
-With `Grid`, defining columns can feel like a static part, but columns and data go strongly hand in hand. We recommend defining a `vaadin-grid` with static attributes in the template and defining columns and data in Java.
+With `Grid`, defining columns can feel like a static part, but columns and data go strongly hand in hand. When using `vaadin-grid` inside a `LitTemplate`, the column configuration and driving of data must to happen from the server side Java code.
```

Comment:
I propose to change `must to happen` into `must happen`.

Reply:
Done